packages/swapper/src/swappers/ThorchainSwapper/utils/getL1quote.ts

```diff
@@ -243,12 +254,16 @@ export const getL1quote = async (
 
             // always use TC auto stream quote (0 limit = 5bps - 50bps, sometimes up to 100bps)
             // see: https://discord.com/channels/838986635756044328/1166265575941619742/1166500062101250100
-            const memo = isStreaming
-              ? quote.memo
-              : addLimitToMemo({
-                  memo: quote.memo,
-                  limit: limitWithManualSlippage,
-                })
+            const memo = (() => {
+              if (!isConnected) return ''
+              if (!quote.memo) throw new Error('no memo provided')
```

Comment:
i.e THOR won't return a memo when we don't pass `destinationAddress` (which is valid, and is what allows us to get quotes without a wallet). But if we're connected and we get a quote without memo, then problems.